pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -247,6 +247,8 @@ nixFlakeBuild() {
                 local k="$1"; shift 1
                 evalArgs+=("$i" "$j" "$k")
                 ;;
+              --impure) # We don't want this in buildArgs
```

Comment:
suggestion: explain *why* we don't want this in buildArgs, perhaps: "impure is only for eval, nix-store doesn't know this flag so don't want it in buildArgs"?

Reply:
wait, it seems you changed wrong occurrence of that comment, or was that intentional? :D

Reply:
Oops. Guess I should be more awake when doing this. Fixed now.

Reply:
ok, LGTM, just waiting for ofborg to finish